### ISSUE TITLE
Accessibility Testing - Add Dashboard Text

### ISSUE THREAD

**Issue by T-T-Nguyen** [StatCan/WellbeingCheckUAT#236]
Placeholder as per request

**Comment by mullhol** [comment]
@Jiandong-Guo Here is the text to be embedded in the dashboard. This should only be visible to readers for the visually impaired, and the mail link needs to be a mailto link. 

The 'Results' section of the Well-being Check contains summaries of your responses to the questionnaire. There is a snapshot of your overall self-rated feelings, and three bar charts that display a summary of your feelings by location, activity, and who you are with. If your screen reader cannot read these results, we would be happy to send them to you in a different format. Please contact Statistics Canada at STATCAN.wellbeingcheck-bilanbien-etre.STATCAN@canada.ca with your name and postal code as entered in the first well-being check you completed.

La section des résultats du Bilan bien-être contient des sommaires de vos réponses au questionnaire. On y trouve un aperçu de vos sentiments généraux autoévalués, ainsi que trois graphiques à barres qui affichent un sommaire de vos sentiments selon votre emplacement, l’activité que vous faites et avec qui vous vous trouvez. Si votre lecteur d’écran n’arrive pas à lire ces résultats, nous pouvons vous les envoyer dans un autre format. Veuillez écrire à Statistique Canada à l’adresse STATCAN.wellbeingcheck-bilanbien-etre.STATCAN@canada.ca en indiquant votre nom et votre code postal comme vous les avez entrés dans le premier Bilan bien-être que vous avez rempli.

CC: @T-T-Nguyen

**Comment by Jiandong-Guo** [comment]
@mullhol 
1) do we need apostrophe ('Result')  in English version?
2) The embedded email link may be a issue. because right now it is just a hidden text of <img> tag, if we want to add a clickable button, first thing we need to separate this link from text and make it to a button, normally the hidden button is not clickable.   Will do some research, and let you know the feasibility.

**Comment by mullhol** [comment]
@Jiandong-Guo Yes, keep the apostrophes.  COMM has reviewed the content and did not flag them for removal, so they should stay. 

Kepp me posted on whether ot not the link will be an issue.

**Comment by Jiandong-Guo** [comment]
@mullhol ,I did some research on this, and found that cannot make it clickable because this string is totally hidden on the screen

**Comment by mullhol** [comment]
Accessibility is being contacted to see if there is another option agreeable to their processes. Subject Matter and CPRD are also investigating if a shorter email could be used. Ticket will be placed on hold until investigation is over.

**Comment by mullhol** [comment]
@T-T-Nguyen Please update this ticket once a decision has been reached so it can either be tested, or Jiandong can complete further work pre-testing

**Comment by Jiandong-Guo** [comment]
@T-T-Nguyen Please see the above text both in English/French, I think it will be modified due to the un-clickable hidden text

**Comment by T-T-Nguyen** [comment]
SMA and Accessibility are aware of the unclickable hidden link. 

Please proceed with the following text from SMA:

The 'Results' section of the Well-being Check contains summaries of your responses to the questionnaire. There is a snapshot of your overall self-rated feelings, and three bar charts that display a summary of your feelings by location, activity, and who you are with. If your screen reader cannot read these results, we would be happy to send them to you in a different format. Please contact Statistics Canada at STATCAN.wellbeingcheck-bilanbien-etre.STATCAN@canada.ca with your name and postal code as entered in the first well-being check you completed.


La section des résultats du Bilan bien-être contient des sommaires de vos réponses au questionnaire. On y trouve un aperçu de vos sentiments généraux autoévalués, ainsi que trois graphiques à barres qui affichent un sommaire de vos sentiments selon votre emplacement, l’activité que vous faites et avec qui vous vous trouvez. Si votre lecteur d’écran n’arrive pas à lire ces résultats, nous pouvons vous les envoyer dans un autre format. Veuillez écrire à Statistique Canada à l’adresse STATCAN.wellbeingcheck-bilanbien-etre.STATCAN@canada.ca en indiquant votre nom et votre code postal comme vous les avez entrés dans le premier Bilan bien-être que vous avez rempli.

**Comment by T-T-Nguyen** [comment]
I've asked SMA to review their text again - We will provide you with the final version accordingly.

**Comment by T-T-Nguyen** [comment]
The 'Results' section of the Well-being Check contains summaries of your responses to the questionnaire. There is a snapshot of your overall self-rated feelings, and three bar charts that display a summary of your feelings by location, activity, and who you are with. If your screen reader cannot read these results, we would be happy to send them to you in a different format. Please send us an email with your name and postal code to infostats@canada.ca (e-mail address must be manually entered).  Please also ensure your name and postal code is the same as what was entered in the first well-being check you completed to submit this request.


La section des résultats du Bilan bien-être contient un résumé de vos réponses au questionnaire. On y trouve un aperçu de vos sentiments généraux autoévalués, ainsi que trois graphiques à barres affichant un sommaire de vos sentiments selon votre emplacement, vos activités, et les personnes avec qui vous êtes. Si votre lecteur d’écran n’arrive pas à lire ces résultats, nous pouvons vous les envoyer dans un autre format. Veuillez nous écrire à l’adresse infostats@canada.ca en indiquant votre nom et votre code postal (l’adresse de courriel doit être entrée manuellement). Veuillez également vous assurer que votre nom et votre code postal sont identiques à ceux que vous avez entrés dans le premier Bilan bien-être que vous avez rempli pour présenter cette demande.

**Comment by Jiandong-Guo** [comment]
Changed, ready to test in next release

**Comment by T-T-Nguyen** [comment]
@Jiandong-Guo sent to Accessibility for review

**Comment by zhuelle** [comment]
This ticket is still with Accessibility to review and confirm.

**Comment by T-T-Nguyen** [comment]
If an update is required it will be completed through #254
